data/management/commands/import_contributors_data.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import json
+from data.models import Contributor
+from django.core.management.base import BaseCommand
+
+IMPORT_URL = 'https://webservices.coala.io/contrib/'
+
+
+class Command(BaseCommand):
+
+    def import_data(self, contributor):
+        login = contributor.get('login', None)
+        name = contributor.get('name', None)
+        bio = contributor.get('bio', None)
+        num_commits = contributor.get('contributions', None)
+        issues_opened = contributor.get('issues', None)
+        reviews = contributor.get('reviews', None)
+
+        try:
+            c, created = Contributor.objects.get_or_create(
+                login=login,
+                name=name,
+                bio=bio,
+                num_commits=num_commits,
+                issues_opened=issues_opened,
+                reviews=reviews
+            )
+            if created:
+                c.save()
+                display_format = "\nContributor, {}, has been saved."
+                print(display_format.format(c))
+        except Exception as ex:
+            print(str(ex))
+            msg = "\n\nSomething went wrong saving this contributor: {}\n{}".format(login, str(ex))
+            print(msg)
```

Comment:
Any reason to save this in a `msg` variable first before printing? Just dilutes the code in my opinion.